pkg/reader/ccip.go

```diff
@@ -625,6 +625,17 @@ func (r *ccipChainReader) GetRMNRemoteConfig(
 	}, nil
 }
 
+func (r *ccipChainReader) isValidAddress(address []byte, contractName string, chain cciptypes.ChainSelector) bool {
+	if len(address) == 0 {
```

Comment:
Check for zero addresses as well? The discovery processor has this helper:
```
// isZero returns true if data is nil or all zeros, otherwise returns false.
func isZero(data []byte) bool {
	for _, v := range data {
		if v != 0 {
			return false
		}
	}
	return true
}
```

Reply:
I don't remember seeing the processor warnings about the empty router though while debugging 🤔. hmmmmm..... maybe I just missed it in the huge log file. Let me remove it so we don't have duplicates for now and keep my eye on it for next couple of PRs.